src/js/letters/containers/AddressSection.jsx

```diff
@@ -208,7 +208,11 @@ export class AddressSection extends React.Component {
       const city = address.city || '';
       const militaryStateCode = address.stateCode || '';
       cityStatePostal = `${city}, ${militaryStateCode} ${zipCode}`;
+    } else {
```

Comment:
Suppose I could have just initialized cityStatePostal in the variable's declaration, but thought it would be easier to understand this way.